app/services/classroom_config.rb

```diff
@@ -0,0 +1,65 @@
+# frozen_string_literal: true
+
+class ClassroomConfig
+  CONFIGURABLES   = %w[issues].freeze
+  CONFIGBRANCH    = "github-classroom"
```

Comment:
We still underscore our constants. E.g. `INVALID_SELECTION = "Invalid repository selection, please check it again."`